private/pkg/thread/thread.go

```diff
@@ -91,19 +90,19 @@ func Parallelize(ctx context.Context, jobs []func(context.Context) error, option
 		select {
 		case <-ctx.Done():
 			stop = true
-			retErr = multierr.Append(retErr, ctx.Err())
+			errs = append(errs, ctx.Err())
 		case semaphoreC <- struct{}{}:
 			select {
 			case <-ctx.Done():
 				stop = true
-				retErr = multierr.Append(retErr, ctx.Err())
+				errs = append(errs, ctx.Err())
```

Comment:
Unlikely, but I think `errs` and above require wrapping with the lock to avoid racing with jobs write below.

Reply:
yea this is a pretty bad miss, this was true before this PR too, will change

Reply:
fixed